toolkits/global/packages/global-ad/scss/10-settings/default.scss

```diff
@@ -0,0 +1,12 @@
+$ad--font-family: $font-family-sans;
+$ad--font-size: 1.4rem;
+$ad--leaderboard-bg-color: #ccc;
```

Comment:
`greyscale(something)`?

Reply:
So James wants to use the same colors on both springer and nature but greyscale(value) outputs different for both brands... so does this become an argument for making greyscale vars the same across brands?

Otherwise I will create settings for springer and nature and try to get the greyscale mixin output close but chance that it won't be exactly the same.

Reply:
I think James had intended to standardise greys across the brands (and maybe ditch the `grayscale()` function in favour of a map of standard greys now we have more shared oversight to stop them getting out of control). You wouldn't have to do that in this PR though! I would stick with the explicit hex values and i'll chat to James about standard grey colours.